src/app/reducers/area.reducer.spec.ts

```diff
@@ -0,0 +1,43 @@
+import { AreaReducer } from './area.reducer';
+import { initialState as AreaInitialState, AreaState } from './../states/area.state';
+import * as AreaActions from './../actions/area.actions';
+import { AreaUI } from './../models/area.model';
+export type Action = AreaActions.All;
+
+describe('AreaReducer', () => {
+  describe('undefined actions', () => {
```

Comment:
@vikram-raj Why a `describe` block inside `describe` block?
The usual pattern is 
```
describe
... it
... it
... it
```